### PR TITLE
Maintenance/update readme

### PR DESCRIPTION
**Pull request by arnupretorius**
## What?
Update to readme to remove outdated information and share more info on Mava's future roadmap.
## Why?
It is important to be transparent and keep users up to date.
## How?
Update main repo readme.

closes #378

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,29 +9,23 @@
...
...constructors as additional modules get added. Our class designs are also overly reliant on wrappers and inheritance which do not scale as well as we would like with increases in system complexity. Furthermore, our original motivation for choosing Tensorflow 2 (TF2) as our deep learning backend was to align with Acme's large repository of RL abstractions and tools for TF2. These were very useful for initially building our systems. But since then, we have found TF2 less performant and flexible than we desire given alternative frameworks. Acme has also affirmed their support of Jax underlying much of the DeepMind RL ecosystem. Therefore, in the coming months, following our stable release, **we plan to rollout a more modular and flexible build system specifically for Jax-based systems.** Please note that all TF2-based systems using the old build system will be maintained during the rollout. However, once a stable Jax release has been made with the new build system, Mava will only support a single DL backend, namely Jax, and we will begin to deprecate all TF2 systems and building support. That said, we will make sure to communicate clearly and often during the migration from TF2 to Jax.
```
Comment (AsadJeewa) [inline README.md:22]: This is very clear regarding the reasons for changes but is this not too much information for the external user?

**Comment by AsadJeewa** [comment]
I really like that we are moving away from Acme. Some important updates to the README

File: README.md [hunk 2]
```diff
@@ -9,29 +9,23 @@
 [![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/instadeepai/Mava/blob/main/LICENSE)
 [![docs][rtd-badge]][documentation]
 
-Mava is a library for building multi-agent reinforcement learning (MARL) systems. Mava provides useful components, abstractions, utilities and tools for MARL and allows for simple scaling for multi-process system training and execution while providing a high level of flexibility and composability.
+Mava is a library for building multi-agent reinforcement learning (MARL) systems. Mava provides useful components, abstractions, utilities and tools for MARL and allows for simple scaling for multi-process system training and execution while providing a high level of flexibility and composability. “Mava” means experience, or wisdom, in Xhosa one of South Africa’s eleven official languages.
```
Comment (RuanJohn) [inline README.md:13]: “Mava” means experience, or wisdom, in Xhosa - one of South Africa’s eleven official languages.

Just a minor change.

File: README.md (hunk 1 above)
Comment (arnupretorius) [reply README.md:22]: I'm very much for being as transparent and open as possible but at the same time willing to hear alternatives and arguments against this. Just note, this discussion is of course already public 😄  So happy to take it offline if needed.

**Review by RuanJohn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AsadJeewa) [reply README.md]: ah yes its easy to forget that

**Review by jcformanek — APPROVED** [review]
Looks good.

**Review by KaleabTessera — APPROVED** [review]
Looks good @arnupretorius , thanks!

**Pull request merged**